### PR TITLE
Bani / Const Op handling for empty and shorter vector values NGTF-2442

### PR DESCRIPTION
**Pull request by bani-intelaipg**

### CODE REVIEW

File: test/python/test_const.py [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+        reason="NGraph will throw error, but TF will fill in zeros")
+    def test_const_empty(self):
+        log = logging.getLogger('test_const_empty')
+        zz = tf.constant([], dtype=float, shape=[2, 3])
+
+        def run_test(sess):
+            log.debug('Invoking sess.run(zz)')
+            return sess.run(zz)
+
+        assert (
```
Comment (shresthamalik) [inline test/python/test_const.py]: instead of skipping the test we could assert on error being thrown?

File: ngraph_bridge/ngraph_utils.h [hunk 2]
```diff
@@ -140,32 +140,38 @@ Status ValuesFromConstNode(const NodeDef& node,
       n_elements *= shape.dim(i).size();
     }
     values->resize(n_elements);
+
+    auto val_lastsaved = 0;
+#define SET_VALS(val_size, val_vec)                        \
```
Comment (shresthamalik) [inline ngraph_bridge/ngraph_utils.h]: can we use a templated function instead?

Comment (bani-intelaipg) [reply ngraph_bridge/ngraph_utils.h]: Restructured the code

**Comment by bani-intelaipg** [comment]
@shresthamalik please review the latest changes before merging.

**Pull request merged**